### PR TITLE
end on destroy

### PR DESCRIPTION
**Pull request by vhmth**
Fixing https://github.com/felixge/node-growing-file/issues/4

### CODE REVIEW

File: lib/growing_file.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ GrowingFile.open = function(path, options) {
 GrowingFile.prototype.destroy = function() {
   this.readable = false;
   this._stream = null;
+  this._ended = true;
```
Comment (vhmth) [inline lib/growing_file.js:52]: Actually I'm wondering if it's more appropriate to override `GrowingFile.prototype.end` or set `this._idleTimeout` to `this._timeout + 1` and then call `this.end();` interally.

**Comment by felixge** [comment]
@vhmth thanks for your patch. I'm no longer using node, so I won't be able to review. However, if you give me your NPM user handle, I'll add you as a pkg owner as well as a contributor on GH, so you can release any patches you deem good yourself :).

**Comment by vhmth** [comment]
Hmm I think you should revert this actually. I took the small silence as you weren't going to maintain this anymore. :-P In the meantime, I've used this module successfully and have figured out how we may want `end` to actually work.

My NPM handle is `vinayh`.

**Comment by paulius005** [comment]
love how those tests read, the wall of setup was totally worth it

red

**Comment by vhmth** [comment]
@paulius005 wrong PR :-P

**Comment by felixge** [comment]
@vhmth added you on npm and gh. Feel free to make and release any changes you need.

> I took the small silence as you weren't going to maintain this anymore

Well, it mostly means that I suck at managing my inbox. But yeah, I'm also not maintaining my old node modules anymore beyond giving commit bits to potential contributors.

**Comment by vhmth** [comment]
Hey Felix - left you a message!

https://www.useloom.com/share/a51d76a90e554e8586f716ff54c93026

**Pull request closed without merging**